pages/mint-nft/index.vue

```diff
@@ -303,6 +391,73 @@ function onISCNFileChange (event: Event) {
   reader.readAsText(file)
 }
 
+async function onClickMintByInputting () {
+  isLoading.value = true
+  step.value = 3
+  const { contentMetadata } = iscnData.value
+
+  const nftClassData = {
+    name: contentMetadata.name,
+    description: contentMetadata.description,
+    symbol: 'CNR',
```

Comment:
What does `CNR` mean here?

Reply:
I don't know... should be from one of the early sample or template